aten/src/ATen/templates/CodegenFunctions.h

```diff
@@ -0,0 +1,30 @@
+// ${generated_comment}
```

Comment:
nit: can we make the name less generic, maybe something like `UnboxingFunctions.h` (similar to how we have a `Functions.h` for the normal unboxed C++ API).

and can you add a comment at the top of the file describing the purpose of the API? (Something about how we code-generate an unboxing API for every native operator).